doc/libghdl/Initialization.rst

```diff
@@ -0,0 +1,21 @@
+.. _LIB:Initialization:
+
+Initialization
+##############
+
+To be able to use `libghdl`, it first must be initialized.
+
+* Why does it must be initialized?
+* What does the initialization do?
+* Is there any use case where someone would want to use libghdl without initialization?
+* If no, then why it does not initialize itself automatically?
+* What functions need to be called during initialization? In Python bindings 3 functions are called:
+
+.. code-block:: python
+
+   libghdl.libghdl_init()
+   libghdl.libghdl__set_hooks_for_analysis()
+
+   # Set the prefix in order to locate the vhdl libraries.
+   libghdl.libghdl__set_exec_prefix(
+       *_to_char_p(dirname(dirname(_libghdl_path)).encode('utf-8')))
```

Comment:
If the comment is not clear enough, please tell me.

But this is 'just' the start of the initialization.  If you look at `vhdl_langsever/workspace.py` you will see the remaining (in `__init__`):
* redirecting errors
* setting flags and options
* end of initialization (load libraries).